.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: '*'
+          node-version: '18'
```

Comment:
gatsby v3 which is tested doesn't work on newer nodes - https://github.com/netlify/netlify-plugin-gatsby/actions/runs/12812982624/job/35725952139#step:9:950